datastore/datastore.go

```diff
@@ -69,6 +78,21 @@ type Client struct {
 // NewClient to detect the project ID from the credentials.
 // Call (*Client).Close() when done with the client.
 func NewClient(ctx context.Context, projectID string, opts ...option.ClientOption) (*Client, error) {
+	client, err := NewClientWithDatabase(ctx, projectID, DefaultDatabaseID, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return client, nil
+}
+
+// NewClientWithDatabase creates a new Client for given dataset and database.  If the project ID is
```

Comment:
Let's format this trailing line -- right now we have two spaces after the period.